File: src/main/resources/output/IFC4X3_java/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcTessellatedItem/IfcIndexedPolygonalFace/IfcIndexedPolygonalFace.java
public class IfcIndexedPolygonalFace extends IfcTessellatedItem {
    public List<IfcPositiveInteger> CoordIndex;

    // === EXTENDED BY ===
    // IfcIndexedPolygonalFaceWithVoids

    // === INVERSE CLAUSES ===
    // ToFaceSet : SET [1:?] OF IfcPolygonalFaceSet FOR Faces
    // HasTexCoords : SET [0:1] OF IfcTextureCoordinateIndices FOR TexCoordsOf
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRepresentationItem/IfcGeometricRepresentationItem/IfcTessellatedItem/IfcIndexedPolygonalFace/IfcIndexedPolygonalFace.vb
Public Class IfcIndexedPolygonalFace Inherits IfcTessellatedItem
    Public CoordIndex As List(Of IfcPositiveInteger)

    ' === EXTENDED BY ===
    ' IfcIndexedPolygonalFaceWithVoids

    ' === INVERSE CLAUSES ===
    ' ToFaceSet : SET [1:?] OF IfcPolygonalFaceSet FOR Faces
    ' HasTexCoords : SET [0:1] OF IfcTextureCoordinateIndices FOR TexCoordsOf
End Class
